smart_campus/app/management/commands/setroles.py

```diff
@@ -0,0 +1,9 @@
+from django.core.management.base import BaseCommand, CommandError
+from app.models import Role
+
+class Command(BaseCommand):
+    help = 'Insert predefined roles into database'
+
+    def handle(self, *args, **options):
+        Role.insert_roles()
```

Comment:
What happens when these roles are already inserted?

Also, changing the file name to `init_roles.py` or `initroles.py` might be better.

Reply:
It's handled in the method.
The method is like insert or update. 
It'll update the permission if there exists a role with the same name

Reply:
link:
https://github.com/rapirent/smart_campus/blob/develop-API/smart_campus/app/models.py#L148